silverpeas/content.md

```diff
@@ -0,0 +1,179 @@
+-	the user domains and the domain authentication definitions in respectively `/opt/silverpeas/properties/org/silverpeas/domains` and `/opt/silverpeas/properties/org/silverpeas/authentication`,
+-	the workflows created by the workflow editor in `/opt/silverpeas/xmlcomponents/workflows`.
+
+Beside these directories, according to your specific needs, custom configuration scripts can be added in the directories `/opt/silverpeas/configuration/jboss` and `/opt/silverpeas/configuration/silverpeas`.
+
+The directories `/opt/silverpeas/log`, `/opt/silverpeas/data`, `/opt/silverpeas/properties`, `/opt/silverpeas/xmlcomponents/workflows`, and `/opt/silverpeas/configuration` are all defined as volumes in the Docker image.
+
+To define a Data Volume Container for Silverpeas, for example:
+
+	$ docker create --name silverpeas-store \
+	    -v silverpeas-data:/opt/silverpeas/data \
+	    -v silverpeas-log:/opt/silverpeas/log \
+	    -v silverpeas-properties:/opt/silverpeas/properties \
+	    -v /etc/silverpeas/config.properties:/opt/silverpeas/configuration/properties \
+	    silverpeas \
+	    /bin/true
+
+Then to mount the volumes in the Silverpeas container:
+
+	$ docker run --name silverpeas -p 8080:8000 -d \
+	    --link postgresql:database \
+	    --volumes-from silverpeas-store \
+	    silverpeas
+
+If you have to customize the settings of Silverpeas or add, for example, a new database definition, then specify these settings with the Data Volume Container:
+
+	$ docker create --name silverpeas-store \
+	    -v silverpeas-data:/opt/silverpeas/data \
+	    -v silverpeas-log:/opt/silverpeas/log \
+	    -v silverpeas-properties:/opt/silverpeas/properties \
+	    -v /etc/silverpeas/config.properties:/opt/silverpeas/configuration/properties \
+	    -v /etc/silverpeas/CustomerSettings.xml:/opt/silverpeas/configuration/silverpeas/CustomerSettings.xml \
+	    -v /etc/silverpeas/my-datasource.cli:/opt/silverpeas/configuration/jboss/my-datasource.cli \
+	    silverpeas \
+	    /bin/true
+
+# Document conversion
+
+Some features in Silverpeas (export, preview, content visualization, ...) requires a document converter. The document conversion is performed in Silverpeas by the program LibreOffice running as a daemon. So, in order to enable and to use these features, you have first to use a Data Volume Container to store all the Silverpeas data and second to run a container embbeding a LibreOffice program running as a daemon. There is no official Docker images of LibreOffice but DockerHub hosts some of unofficial images of it ([xcgd/libreoffice](https://hub.docker.com/r/xcgd/libreoffice/) for example).
+
+Once a Data Volume Container created for Silverpeas as explained in the section above, you have to link it with the Docker image running LibreOffice as a daemon in order the program have access the documents to convert:
+
+	$ docker run --name libreoffice -d \
+	    --volumes-from silverpeas-store \
+	    xcgd/libreoffice
```

Comment:
The Silverpeas image still contains libreoffice, right?  Could that be invoked here instead of depending on a non-official image? (https://github.com/docker-library/official-images/blob/20311d4ee3ecf4f832e405089f06ce83dd1972e5/README.md#repeatability)

ie, something like:
```console
$ docker run --name libreoffice -d ... silverpeas libreoffice --nologo --norestore --invisible --headless --accept='socket,host=0,port=8997,tcpNoDelay=1;urp;'
```